.github/workflows/build-firmware-runners.yaml

```diff
@@ -0,0 +1,335 @@
+name: FW-new
+
+on: [push, pull_request]
+
+jobs:
+  build-firmware:
+    runs-on: firmware-ci
+    
+    strategy:
+      # Let all builds finish even if one fails early
+      fail-fast: false
+      matrix:
+        # What boards should we build for? In the 'include' section below,
+        # set up what each of these boards needs to build.
+        #
+        # see also gen_config where we have a similar list for all boards!
+        # see also build-primary-bundle where default/primary bundle is build separately
+        #
+        build-target: [
+            hellen72,
+            hellen81,
+            hellen88bmw,
+            hellen-nb1,
+            hellenNA6,
+            hellen128,
+            hellen121vag,
+            hellen121nissan,
+            cypress,
+            frankenso_na6,
+            kinetis,
+            mre_f4,
+            mre_f4_recovery,
+            mre_f4_hardware_QC_special_build,
+            mre_f7,
+            prometheus_405,
+            prometheus_469,
+            proteus_f4,
+            proteus_f4_hardware_QC_special_build,
+            proteus_f7,
+            proteus_f7_hardware_QC_special_build,
+            proteus_h7,
+            proteus_legacy,
+            stm32f767_nucleo,
+            stm32h743_nucleo,
+            subaru_eg33_f7,
+            atlas,
+        ]
+
+        include:
+          # Board configurations
+          - build-target: hellen72
+            folder: hellen/hellen72
+            ini-file: rusefi_hellen72.ini
+
+          - build-target: hellen81
+            folder: hellen/hellen81
+            ini-file: rusefi_hellen81.ini
+
+          - build-target: hellen88bmw
+            folder: hellen/hellen88bmw
+            ini-file: rusefi_hellen88bmw.ini
+
+          - build-target: hellen-nb1
+            folder: hellen/hellen-nb1
+            ini-file: rusefi_hellen-nb1.ini
+
+          - build-target: hellenNA6
+            folder: hellen/hellen64_miataNA6_94
+            ini-file: rusefi_hellenNA6.ini
+
+          - build-target: hellen128
+            folder: hellen/hellen128
+            ini-file: rusefi_hellen128mercedes.ini
+
+          - build-target: hellen121vag
+            folder: hellen/hellen121vag
+            ini-file: rusefi_hellen121vag.ini
+
+          - build-target: hellen121nissan
+            folder: hellen/hellen121nissan
+            ini-file: rusefi_hellen121nissan.ini
+
+          - build-target: hellen154hyundai
+            folder: hellen/hellen154hyundai
+            ini-file: rusefi_hellen154hyundai.ini
+
+          - build-target: cypress
+            folder: hellen
+            ini-file: rusefi_hellen_cypress.ini
+
+          - build-target: frankenso_na6
+            folder: frankenso
+            ini-file: rusefi_frankenso_na6.ini
```

Comment:
We need to at least move ini file into a separate dictionary not to duplicate it?